doc/Usage.md

```diff
@@ -224,6 +224,11 @@ $pages = $pdf->getPages();
 $mediaBox = [];
 foreach ($pages as $page) {
     $details = $page->getDetails();
+    // The page could haven't the Mediabox, then we get this information from document
```

Comment:
The comment doesn't reflect the code in my opinion. Here is an alternative:

```suggestion
    // If Mediabox is not set in details of current $page instance, get details from the header instead
```

Reply:
Thanks! English is not my main language.